apps/govquests-api/rails_app/app/read_models/questing/on_quest_created.rb

```diff
@@ -1,7 +1,7 @@
 module Questing
   class OnQuestCreated
     def call(event)
-      slug = event.data[:display_data][:title].downcase.tr(" ", "-")
+      slug = event.data[:display_data][:title].downcase.tr(" ", "-").delete(":")
```

Comment:
`#could` use `.parameterize` instead:
```suggestion
      slug = event.data[:display_data][:title].parameterize
```